docs/surveys/2020-jupyterlab-survey.md

```diff
@@ -0,0 +1,363 @@
+    - ❗I am not performing ML/statistical tasks.
+    - Regression (predict a numeric output).
+    - Classification (predict a categorical output).
+    - Generative/ auto-encode (create new data based on existing data).
+    - Reinforcement learning (actions that maximize a reward)
+    - Dimensionality reduction (PCA, K-Nearest Neighbors)
+    - Feature engineering (importance, extraction, selection, permutation).
+    - Natural language processing.
+    - Graph data science.
+    - Outlier detection.
+    - Other (please specify)
+    
+    
+### Visualization
+
+12. What tools does your team use to create dashboards tools? (pick up to 3)
+    - ❗My team doesn’t create dashboard tools.
+    - Write our own in HTML & JS.
+    - R Shiny.
+    - Kibana.
+    - Dash-Plotly.
+    - Voila.
+    - Tableau.
+    - Looker.
+    - Klipfolio.
+    - Google Data Studio.
+    - Spotfire.
+    - Grafana
+    - Other (please specify)
+
+13. Do you experience these problems with visualization in Jupyter?
+    - No built-in UI for creating charts.
+    - Can't publish my charts as web-based dashboards.
+    - Poor/ buggy support for my plotting tool.
+    - Displaying highly dimensional data (array of array of arrays, too many rows/ columns to fit on screen).
+    - Lacking templating support (Jinja2)
+
+    For each of the items above, specify:
+    - Not a problem for me.	
+    - Trivial.	
+    - Minor.	
+    - Major.	
+    - Critical.	
+    - N/A - skip, don't know.
+
+### Scale
+
+14. How do you scale and schedule your workloads? (up to 4)
+    - They run just fine on my local machine 😃.
+    - ❓I need to scale, but don't know how.
+    - On-premise HPC/ data center.
+    - Cloud server (e.g. EC2)
+    - Cloud service for ML/ AI (e.g. AWS SageMaker)
+    - Spark cluster (e.g. similar Hadoop, Dask).
+    - K8s cluster (e.g. Mesos, Swarm, Slurm).
+    - Jupyter Enterprise Gateway.
+    - Jupyter BinderHub.
+    - Quantum (e.g. D-Wave).
+    - Horovod.
+    - Kubeflow.
+    - Elyra.
+    - Workflow - Snakemake.
+    - Workflow - Papermill
+    - Workflow - Nextflow, WDL, CWL.
+    - Workflow - Airflow.
+    - Workflow - cloud service (e.g. AWS Batch)
+    - Cloud query service (AWS Presto or Athena)
+    - Other (please specify)
+
+15. Do you experience these problems with scale in Jupyter?
+    - Figuring out how to schedule batch execution of notebook-based jobs.
+    - Don’t have the budget for more scalable environment/ cloud services.
+    - Haven’t divided longer notebooks into multiple, modular notebooks.
+    - Not persisting the outputs of a notebook.
+    - ML training jobs take too long.
+    - Can't call code/ modules from other notebooks.
+    - Managing Spark dependencies (Java) and connections.
+
+    For each of the items above, specify:
+    - Not a problem for me.	
+    - Trivial.	
+    - Minor.	
+    - Major.	
+    - Critical.	
+    - N/A - skip, don't know.
+
+### Collaboration
+
+16. With how many other people are you collaborating?
```

Comment:
collaborating on 🙃 ?

Reply:
On notebooks

Reply:
agree, let's make it more explicit about "notebooks"

Reply:
☑️ changes integrated: "When it comes to working on notebooks in a team setting, with how many other people are you collaborating?"